tests/accessor/accessor_common.h

```diff
@@ -0,0 +1,450 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Common functions for the accessor tests.
+//
+*******************************************************************************/
+
+#ifndef SYCL_CTS_ACCESSOR_COMMON_H
+#define SYCL_CTS_ACCESSOR_COMMON_H
+
+#include "../../util/sycl_exceptions.h"
+#include "../common/common.h"
+#include "../common/type_coverage.h"
+#include "../common/value_helper.h"
+
+#include "catch2/catch_test_macros.hpp"
```

Comment:
I think we usually use `<angle brackets>` for third-party includes!

Reply:
Actually, looks like it would be better to use the following:
```
#include "../common/common.h"
// instead of <catch2/catch_test_macros.hpp>
```
and
```
#include <catch2/catch_template_test_macros>
#include "../common/macros.h"
// instead of <catch2/catch_template_test_macros.hpp>
```
because of `#undef FAIL` within [../common/macros.h](https://github.com/KhronosGroup/SYCL-CTS/blob/0a5d5d689124edcc2280a0d7d36d2e16513904f5/tests/common/macros.h#L16)
@psalz, @andreyromanof   what do you think?

Reply:
You're right, including `<catch2/catch_test_macros.hpp>` is superfluous in this case, just including "`../common/common.h"` should be sufficient (as that transitively includes `macros.h`). Also the custom `FAIL` macro is actually not really needed in this case, it only exists for backwards compatibility with the old `FAIL` macro that received a `logger`.

Reply:
Removed extra includes and added angle brackets to catch header `catch2/matchers/catch_matchers.hpp` include